### ISSUE TITLE
Identity attributes not passed in 7.1.2

### ISSUE THREAD

**Issue by nijel** [IdentityPython/pysaml2#857]
## Code Version

This appeared in 7.1.2

## Expected Behavior

The assertion should contain identity attributes.

## Current Behavior

No attributes are present.

## Possible Solution

The following patch does address the issue for me:

```patch
--- assertion.py.orig	2022-04-06 16:31:06.186251104 +0200
+++ assertion.py	2022-04-06 16:31:09.818258405 +0200
@@ -89,7 +89,7 @@
         name_format = attr.get('name_format')
         name = attr.get('name')
         friendly_name = attr.get('friendly_name')
-        local_name = get_local_name(acs, name, name_format) or friendly_name
+        local_name = get_local_name(acs, name, name_format)
 
         _fn = (
             _match(local_name, ava)
```

I have no clue what was an intention when introducing this in af48058d1ed605669f5f47a0cac0e5a146a1fb44, so it might be breaking something else...

Content of these variables for me (this is coming from [omniauth](https://github.com/omniauth/omniauth-saml/blob/f2a7a8459a77f9901750d43460f33d5a095ac28a/lib/omniauth/strategies/saml.rb#L17-L22)):

```python
name = 'last_name', 
friendly_name = 'Family name'
ava = {"last_name": "Name"}
```

## Steps to Reproduce

1. Pass identity attribute which is not resolved by the acs
2. See it gets filtered

**Comment by c00kiemon5ter** [comment]
Hello, sorry for this breakage and thanks for the report.

This should now be fixed by f36b06aa2a8aeb41394139d0fa2cf20b59f41dd9.

I will try to resolve/merge some more issues and will prepare a new release with this fix.

**Comment by c00kiemon5ter** [comment]
closed by `v7.2.0`
